### DISCUSSION TITLE
Awesomeness Directory

### DISCUSSION THREAD

**Discussion by indiebio** [omigroup/omigroup#489] [Show and tell]
These are some of the projects our members are involved in. While we cannot endorse these or guarantee that they align to our values, we would like to shine a light on them. They are listed in no particular order. This is a living document, there is no deadline to adding your initiative. This serves as a reference directory; we may highlight initiatives on social media, but will check in with you first.

Please add your project, content channel, whatever you want to highlight, as a comment. Please include:

1. the project name
2. a link to more info 
3. a short description, do you see this linking with the metaverse? If so, how? 
4. who to contact
5. optionally, you can also add your ask - what do you need help with? Being open to funding, for example
6. image

We reserve the right to edit or remove posts to maintain community standards.

**Comment by indiebio** [comment]
Project Name: **Mirror Worlds** (casual game)
no link yet, idea stage
 Alternative name: "they don't want you to be free" 
The base landscape is an unaltered, curated low poly version of the physical world. Base bubbles of players are placed on this landscape, at a location of the players' choosing. The link to the metaverse is the game architecture in how the game is built to incorporate interactive physical world elements, and a related platform where the game assets and infrastructure could be used for physical world knowledge mapping, similar to OpenStreetMaps.

The game contains challenges that happen in mirror worlds, where different aspects of the physical world is mirrored or distorted, and where elements of the physical world can be modified or interacted with to solve the challenges.

Home base is a bubble, where you can do whatever you want. It's safe, so for someone like me who doesn't want to do combat i can just do my breeding game or farm sim or whatever.

Inside the base bubble the player can select a target area that does not need to relate to the physical location on the map, it is a fresh instance, but limited area/resolution. The player can choose to include the whole Earth in their bubble but at a low resolution, or a small area at high resolution, for example. The elements that are altered in the challenges, for example gravity, height distortions, rainfall etc can be altered by choice by the player inside the base bubble. 

In this way players may want to co-locate on the base layer to be able to find each other and invite friends in, but within their home base bubbles choose a different area of the world they would like to play in.

Contact: @indiebio

**Comment by PabloFMM** [comment]
Project name: Numen Games (Numinia)
Link to more info: https://numen.games/ & https://github.com/numengames

Description: A Template to Reimagine Organizations

Numen Games aim to create a decentralized, living internet where players engage in a gamified structural framework that blends education, exploration, and creative collaboration. Organized into distinct districts aligned with specialized factions, Numinia leverages the power of decentralized knowledge and continuous innovation to keep the experience fresh and engaging. This strategy fosters a community where players are not just participants but co-creators of an ever-evolving narrative, ensuring that Numinia remains a cutting-edge and inclusive space for learning and adventure

Contact: pablofm@numengames.com

**Comment by zodiepupper** [comment]
project name: [barkvr](https://github.com/zodywoolsey/barkvr) (in development, builds available)

Completely open collaborative XR creation tool using godot. the goal is to build a fully network synchronized engine that exposes every feature available in the godot engine. we use the matrix protocol for our user management and for signaling for webrtc sessions.

paxos resolution for sessions, documented interoperability format for session data channels, built in browser, end to end encryption, session process isolation, physics based avatar ik for arbitrary joint chains, and distributed file hosting systems are planned and some are coming very soon.

users should be able to make whatever they want, import whatever they want, and export whatever they want in real time with their friends or colleagues.

contact:
matrix:
@zodiepupper:matrix.pupper.dev
email:
contact@pupper.dev
everywhere else:
zodiepupper

**Comment by indiebio** [comment]
TODO:
Virtual Worlds Museum
Solo's memotar
MTV
jimmy
IPSME
V-sekai
